Packs/GitHub/Integrations/GitHub/GitHub.yml

```diff
@@ -2586,11 +2588,27 @@ script:
     - contextPath: GitHub.PR.ChangedFiles
       description: The number of changed files in the pull request.
       type: Number
-  dockerimage: demisto/pyjwt3:1.0.0.4946
-  subtype: python3
+  - arguments:
+    - default: false
+      description: Repository owner.
+      isArray: false
+      name: owner
+      required: true
+      secret: false
+    deprecated: false
+    description: Get github actions workflows usage of private repositories.
```

Comment:
is this specific to private repositories? what about the public ones?

Reply:
Yes, specific for private repos. Public repos Github actions are free and ```/repos/:owner/:repo/actions/workflows/:workflow_id/timing``` endpoint is not returning any data for public repos